### DISCUSSION TITLE
FREE/PAID functionality (PDF, ...)

### DISCUSSION THREAD

**Discussion by QVinto** [YetiForceCompany/YetiForceCRM#15671] [General] [answered]
Captera.com>
"YetiForce helps companies best utilize free professional open source CRM to lower costs and improve performance throughout the entire communication structure. We offer an opportunity to divert the costs of expensive licenses towards custom built systems tailored exactly to your specifications."

**Could you be so kind and tell us at least 1 year ahead what free functionality will be paid in future?**

Mail  was free now it is paid - i had to disable funciton on some installations
Auto assignment owner was free now its paid - i had to disable all funcitons on some installations
**PDF is free - gitdeveloper shows that new version will be paid.**
![image](https://user-images.githubusercontent.com/4982986/151415800-e0fb7189-5ead-48af-b8d3-5cbdf1079df7.png)
...

...!!!!

I have one installation on yetiforce cloud. 2 side installations. We are using PDF and from next version i will have that f*cking note every time when we login "you are using paid functionality, please make a payment" ??? 

Are you kidding me? Now we will have to pay for it. (BTW this PDF functionality has sooooo many bugs (watermark, tables, header size, font sizes, ...) and you are going to ask money for it...)

What next? paid all modules? paid permissions setup? paid dashboard? paid login page? YF cloud had 750GB storage, now 100GB. When I started to use yetiforce 4 years ago you wrote everywhere that this is free and everyone else have this functionality as paid module. I saw some discusions where you compared all modules and funcions that are free against everyone else.

I get it that all new functionality will be paid, but it is not fair when you make some functionality that it was not annouced before as paid and after years you just decide that it will be paid from next update.

Great approach. 
We are free and opensource but now we are not so free as we were before, just stop using what you used to working with or just pay.

**Comment by bpabiszczak** [comment, marked as answer]
In my opinion, your view of the changes we make is nowhere near the truth. 

You said: "Mail was free now it is paid"

In my opinion, the mail functionality is free. By default the system allows you to install the Roundcube email client, and once you install it you can use it with no limitations. I have no clue why you’re spreading misinformation that the email is paid when it isn’t. Additionally, we released an Outlook addon that is a paid functionality, but using Outlook is not necessary. If you want to use it and you can afford Outlook, then you can also afford the addon. So I have no idea what your deal with paid email is. 

You said: "Auto assignment owner was free now its paid" 

The addon was being used by one our client only and the functionality needed to be expanded in order to be more useful. Due to the fact that the addon is not relevant for 99% of the system users, and on top of that it’s difficult to develop and maintain, we decided to turn it into a paid functionality. Someone who automates the system to this extent can surely afford a few € per month to pay for the addon. If that person can’t afford it - no problem at all, they can still create workflows or custom scripts that will allow them to expand or automate whatever processes they want. 

You said: "PDF is free - gitdeveloper shows that new version will be paid"

The new version will have two PDF libraries, the one we have now that is not ideal as it doesn’t support some tags, and another one based on Chromium PDF. The new mechanism unfortunately requires proper server configuration (the addon won’t most likely work on most hostings) and this one will be the paid one. So the new YetiForce CRM version will have two PDFs, the one available currently, and the new one - PDF Premium. We even changed the free version’s license to MIT (https://github.com/YetiForceCompany/YetiForcePDF/blob/developer/LICENSE) to make it easier for the community to develop it. Unfortunately, developing PDF is extremely time consuming and requires certain programming skills, so a professional PDF can’t be free - it has to be able to return the investment. 

**About the rest of the snarky comments you sent our way:**

YetiForce has an open license that allows you to use, customize, and modify the system practically with no limitations (there are some small limitations that aren’t relevant for 99% of companies). It doesn’t mean that all addons will be free. Currently there are over 80 user modules and over another 80 admin panels, as well as dozens of different mechanisms and tools. The paid extras account for maybe 2 ~ 3% of the whole system - this isn’t a lot. 

Saying "I get it that all new functionality will be paid" is just a lie. Around 90% of all the changes are free, this is how it’s always been, how it is now, and how it will be in the future.
Our main goal, however, is to diversify our income. We don’t want services to be our only source of money, because it’s not reliable (especially in pandemic times), so we have to create paid extensions or offer cloud to our clients, to spread our revenue source over many different channels. Paid addons are usually created when a client can only afford a small part of the entire cost of a new functionality, and we have to contribute using our resources to create the addon. If you want to order an extension and can afford the full price for creating it - good news, we will happily add it to the system for free.    

Please show me any other system that has all these development and customization options and is as flexible as YetiForce CRM. To make it easier for you, you can skip the paid functionalities, I’m curious to see how many alternatives (even paid ones) you can find that will provide you with comparable freedom and flexibility. It’s easy to criticize, but much harder to create something on your own with a limited budget. We’re a small team, we give it all we can, the rest is up to the community. 

As far as the roadmap is concerned - the changes result mainly from the current needs of our clients. We don’t know what changes we’ll introduce in a few days or weeks. Whether or not we create a new extension depends on a current need, not on a plan we had. Our clients design the system, they’re the ones who order new changes and they’re the ones who decide what gets added into the system. We are not able to predict 2 months ahead, not to mention 12 months. For us 12 months ahead is as if someone asked you what you’ll be doing 5 years from now. 

The more products and services our community purchases, the more resources we can devote to creating free stuff. Appreciate what you have, don’t ask how much more we’ll give you for free, instead try contributing something. It’s up to the community how the system will develop. Ask yourself these questions: 

Did you like us or wrote a positive review on any of the following sites: 

1. https://sourceforge.net/projects/yetiforce/
2. https://sourceforge.net/software/product/Yetiforce/ 
3. https://www.capterra.com/p/159123/Yetiforce/
4. https://www.getapp.com/customer-management-software/a/yetiforce/
5. https://www.gartner.com/reviews/market/sales-force-automation/vendor/yetiforce/product/yetiforce-crm
6. https://www.g2.com/products/yetiforce-yetiforce-crm/reviews
7. https://www.softwareadvice.com/crm/yetiforce-profile/
8. https://www.softaculous.com/apps/erp/YetiForce 

Did you tell us about the good things in the system that you use? 
Did you tell us what you are willing to pay for so that we can know better what our community thinks (unless you prefer that we track what you do in the system;) )?
Do you recommend us on social media? Czy polecasz nas w social media. 
And so on

There is a number of things that help us, which don’t require any financial contribution from the other party. However, if your company can afford it, you can always pay for a solution to someone else’s problem, or pay us to create an extension that everybody could use. If everyone who uses the system paid us 1€ per month we would have huge development resources.
